src/main/java/pages/Header.java

```diff
@@ -0,0 +1,47 @@
+package pages;
+
+import base.PageBase;
+import org.openqa.selenium.NoAlertPresentException;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.testng.Assert;
+
+public class Header extends PageBase {
+    public Header(WebDriver driver) {
+        super(driver);
+    }
+
+    @FindBy(xpath = ".//a[@role='button' and contains (@title, 'Account')]")
```

Comment:
Class fields should be placed before ctor in the class according to Java Code Convention